### PR TITLE
SDIT-1672 Add support for offender / offender address phone events

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/service/xtag/XtagContent.kt [hunk 1]
```diff
@@ -52,8 +52,6 @@ class XtagContent(private val map: Map<String, String>) {
   val p_direction_code by m
   val p_escort_code by m
   val p_event_date by m
-  val p_old_alert_date by m
-  val p_old_alert_time by m
```
Comment (mikehalmamoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/service/xtag/XtagContent.kt:56]: Unused

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/model/OffenderEvent.kt [hunk 2]
```diff
@@ -349,6 +349,26 @@ class OffenderBookingReassignedEvent(
...
+class OffenderPhoneNumberEvent(
+  eventType: String?,
+  eventDatetime: LocalDateTime?,
+  nomisEventType: String?,
+  offenderIdDisplay: String?,
+  offenderId: Long? = null,
+  val addressId: Long? = null,
+  val phoneId: Long?,
+  val phoneType: String?,
+  val phoneNo: String?,
```
Comment (petergphillips) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerevents/model/OffenderEvent.kt:362]: perhaps better not to include this in the event? Don't really want the telephone number being passed around if not needed

**Review by petergphillips — DISMISSED** [review]
LGTM

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**